editor/editor_properties.h

```diff
@@ -31,13 +31,17 @@
 #ifndef EDITOR_PROPERTIES_H
 #define EDITOR_PROPERTIES_H
 
+#include "core/math/expression.h"
+#include "core/os/input.h"
 #include "editor/create_dialog.h"
 #include "editor/editor_file_system.h"
 #include "editor/editor_inspector.h"
 #include "editor/editor_spin_slider.h"
 #include "editor/property_selector.h"
 #include "editor/scene_tree_editor.h"
+#include "editor_scale.h"
```

Comment:
Apart from `line_edit.h`, all those added `#include`s don't seem necessary here.

Reply:
Not all of them are actually used, but the program needs the following includes in order to compile:
#include "core/math/expression.h"
#include "core/os/input.h"
#include "editor/editor_spin_slider.h"
#include "editor/property_selector.h"

I edited the pull request and squashed the commits that eliminate the unnecessary includes.